### ISSUE TITLE
Conflicting indentation rules

### ISSUE THREAD

**Issue by pmrcunha** [trustpilot/node-eslint-config#32]
The rule `react/jsx-closing-bracket-location` conflicts with `indent`, when listing props one in each line.

Example:
Triggers `indentation` error:
```jsx
<Text
              id="AppName.helloFramework"
              interpolations={{ FRAMEWORK: framework}}
              tag={placeholderTags}
              />
```

Triggers `react/jsx-closing-bracket-location` error:
```jsx
<Text
              id="AppName.helloFramework"
              interpolations={{ FRAMEWORK: framework}}
              tag={placeholderTags}
/>
```
or
```jsx
<Text
              id="AppName.helloFramework"
              interpolations={{ FRAMEWORK: framework}}
              tag={placeholderTags} />
```

**Comment by DanMeakin** [comment]
I think that setting the `react/jsx-closing-bracket-location` rule to `tag-aligned` might be a good solution to the problem. 

It seems that it is the `prop-aligned` value which is conflicting with the `indentation` rule, and that `tag-aligned` might be a suitable solution to this. I don't personally see any problem with `tag-aligned` as a setting, and it is probably more important to resolve this than to retain the existing rule.

**Comment by madsgodvinjensen** [comment]
I'm with Dan here.

**Comment by madsgodvinjensen** [comment]
After talking to Dan, we agreed to use the `after-props` style.

**Comment by madsgodvinjensen** [comment]
Fixed in release [v3.2.1](https://github.com/trustpilot/node-eslint-config/releases/tag/v3.2.1)
